sklearn/hpsearch/gke_parallel.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
This is the file that I want a lot more documentation for - docstrings especially.

Reply:
I must say, though, I find the code very readable. Nice job!

Reply:
docstrings required in all code files.